Show contributers avatars in contributers tableView

mahmoudfotta opened the pull request:
*add avatarUrl to contributer struct
*decode contributer struct using codingkeys
*add avatarImageView to ContributerCell
*send avatarURL to avatarImageView to load image

mahmoudfotta committed:
Show contributers avatars in contributers tableView

*add avatarUrl to contributer struct
*decode contributer struct using codingkeys
*add avatarImageView to ContributerCell
*send avatarURL to avatarImageView to load image

Yonodactyl commented:
@kgellci Any chance you can check this?

kgellci commented:
Sorry, missed this, looking now.

kgellci commented on Area51/Sources/ContributionsController.swift:
```diff
@@ -32,17 +32,19 @@ class ContributionController: UITableViewController {
         cell.contributionsLabel.text = "\(contributors.contributions) Contributions"
 
+        cell.avatarImageView.isHidden = contributors.avatarURL == nil
+        if let avatarURL = URL(string: contributors.avatarURL ?? "") {
```
You can replace this if else statement with
`cell.avatarImageView.url = contributors.avatarURL.flatMap(URL.init)`

kgellci commented on Area51/Sources/ViewModels/ContributorModel.swift:
```diff
@@ -5,6 +5,20 @@ struct Contributor: Decodable {
+        case avatarURL = "avatar_url"
+    }
+
+    init(from decoder: Decoder) throws {
```
can delete this entire init block, the coding should just work with the CodingKeys specified above.

kgellci commented:
@mahmoudfotta Thanks for adding, I left a couple comments, should be good to go once those are resolved.

mahmoudfotta committed:
Fix pullRequest comments

*Remove init from contributorModel
*User flatmap with avatarUrl instead of if else

mahmoudfotta commented:
fixed comments and pushed @kgellci

kgellci reviewed (approved):
Awesome! Feel free to update the changelog with this change.
